### DISCUSSION TITLE
Change request - device "hide LED"

### DISCUSSION THREAD

**Discussion by misa1515** [emsesp/EMS-ESP32#2194] [Ideas]
### DESCRIPTION

_A clear and concise description of what the problem is or the change requested._

### REQUESTED INFORMATION

_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Searched the issue in [issues](https://github.com/emsesp/EMS-ESP32/issues)
- [x] Searched the issue in [discussions](https://github.com/emsesp/EMS-ESP32/discussions)
- [not relevant] Searched the issue in the [docs](https://emsesp.org/Troubleshooting/)
- [x] Searched the issue in the [chat](https://discord.gg/3J3GgnzpyT)
- [not relevant] Provide the System information in the area below

### TO REPRODUCE

In Settings / Aplication settings is "Hide LED" - working correctly.

Please extend functionality of "Hide LED" switch OFF in defined time (e.g. for switch off the LED during night).

Thank you in advance.

### EXPECTED BEHAVIOUR

Extend functionality of "Hide LED" switch OFF in defined time

### SCREENSHOTS

An example attached (from other application).
![Vzor](https://github.com/user-attachments/assets/c9098d21-dd95-4115-b923-d11812b9d329)

Vždy zapnuté = always ON
Vždy vypnuté =  always OFF
Naplánované vypnutie = Planned switch OFF
Od = From
Do = To
Nasledujúci deň = Next day

**Comment by proddy** [comment]
This is a good case for #2182, where I want to be able to control on/off some of the settings and services. We can extend to do the LED, and then you can use the EMS-ESP Scheduler to automate it. That way the control is with the user, since not everyone is using NTP or has EMS-ESP connected up to the internet.
